### ISSUE TITLE
Problems when updating an AutoscalingConfig or Group via Terraform - IM0034699039 

### ISSUE THREAD

**Issue by CHS-OTC** [opentelekomcloud/terraform-provider-opentelekomcloud#1233]
I noticed a change in the Terraform Provider that makes it impossible for me to make changes to an AutoscalingConfig or Group via Terraform.

The previous behavior was something like this:
- Change to AutoscalingConfig (e.g. instance_config image) -> start terraform
- Terraform creates a new AutoscalingConfig parallel to the old one
- Terraform sets the scaling_configuration_id in the AutoscalingGroup to the ID of the new AutoscalingConfig
- Terraform deletes the old AutoscalingConfig

Current behavior:
- Change to AutoscalingConfig (e.g. instance_config image) -> start terraform
- Terraform tries to change the AutoscalingConfig directly
Plan:
```hcl
  # module.worker_autoscaling_group.opentelekomcloud_as_configuration_v1.worker_as_config will be updated in-place
  ~ resource "opentelekomcloud_as_configuration_v1" "worker_as_config" {
        id                         = "e8f84bc9-a47e-4675-b17b-579313fd3e71"
        # (2 unchanged attributes hidden)

      ~ instance_config {
          ~ image           = "075ea869-bcc5-4c19-a55e-886de2b227ef" -> "1030abd2-f6f1-4e9c-8173-040a93900423"
            # (5 unchanged attributes hidden)

            # (5 unchanged blocks hidden)
        }
    }
```

Error Message:
```
module.worker_autoscaling_group.opentelekomcloud_as_configuration_v1.worker_as_config: Modifying... [id=e8f84bc9-a47e-4675-b17b-579313fd3e71]
╷
│ Error: doesn't support update
│
│   with module.worker_autoscaling_group.opentelekomcloud_as_configuration_v1.worker_as_config,
│   on ../modules/worker_autoscaling_group/main.tf line 31, in resource "opentelekomcloud_as_configuration_v1" "worker_as_config":
│   31: resource "opentelekomcloud_as_configuration_v1" "worker_as_config" {
│
```

terraform_0.15.5 version
Terraform v0.15.5
on linux_amd64
+ provider registry.terraform.io/hashicorp/dns v3.2.0
+ provider registry.terraform.io/hashicorp/template v2.2.0
+ provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.24.1

If I adjust the old behavior manually:
- Temporarily create AS config in parallel
- Change the AS group to the id of the temporary AS config
- Delete primary AS config
- Create a new primary AS config identical to the temporary AS config
- Change the AS group to the id of the primary AS config
- Delete temporary AS config
Everything works so far.

**Comment by lego963** [comment]
@CHS-OTC can you ask to update provider release version?
We had same issue earlier and fixed it in [v1.24.3](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/blob/devel/CHANGELOG.md#1243-june-30-2021)

Refers to: #1149

**Comment by lego963** [comment]
Reopen if it is relevant on `latest` provider version.
